API/src/validators/validateOrders.js

```diff
@@ -0,0 +1,143 @@
+import { param, body, validationResult } from 'express-validator/check';
+import { sanitizeBody } from 'express-validator/filter';
```

Comment:
I don't think you allowed to use a third-party validator (package) to do this. Please do whatever validation is needed with your own `custom` code.

Reply:
Ok. I will remove that but I won't be able to sanitize user's inputs. Isn't that necessary?

Reply:
You should be able to sanitize with your own custom code.